src/js/main.js

```diff
@@ -124,7 +157,7 @@
 
     var openEditorWidget = function openEditorWidget(button, action) {
         if (this.editor_widget == null) {
-            this.editor_widget = mp.mashup.addWidget('CoNWeT/json-editor/1.0', {refposition: button.getBoundingClientRect()});
+            this.editor_widget = mp.mashup.addWidget('CoNWeT/json-editor/1.0', {refposition: this.defaultEditorPos});
```

Comment:
And revert this change 😉.

Reply:
better, use directly the button without calling `getBoundingClientRect`

Reply:
Fixed in 433f4c6.

json-editor static version updated to v1.0.1